pandas/tests/io/test_parquet.py

```diff
@@ -1021,6 +1021,43 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    def test_read_use_nullable_types_pyarrow_config(self, pa, df_full):
```

Comment:
This is failing in the min versions build

Reply:
Thanks, fixed